jekyll/_cci2/hello-world.md

```diff
@@ -29,7 +29,7 @@ This document describes how to configure your project to run on CircleCI 2.0.
           - checkout
           - run: echo "hello world"
     ```
-The image defines the execution environment for your build. The steps check out the code in the project directory and run the `echo` command.
+The first image listed defines the execution environment for your build. So, this first image is where your 2.0 build will run because it is used for the primary container. The steps check out the code in the project directory and run the `echo` command.
```

Comment:
Suggest change "So, this first image is where your 2.0 build will run because it is used for the primary container." to: "The primary container, where your 2.0 build commands run, is created from this image."